src/components/dataset/DatasetScreen.js

```diff
@@ -125,22 +137,41 @@ class DatasetScreen extends Component {
               </Typography>
             </Grid>
             <Grid item xs={6}>
-              <Typography variant="h5">{t('Content')}</Typography>
-              <JSONFileEditor
-                size={datasetSize}
-                id={datasetId}
-                content={datasetContent}
-                collapsed={2}
-                editEnabled
-              />
-            </Grid>
-            <Grid item>
-              <Typography variant="h5">{t('Information')}</Typography>
-              <DatasetInformationTable
-                content={datasetContent}
-                id={datasetId}
-              />
+              <Grid container alignItems="center" spacing={2}>
+                <Grid item>
+                  <Typography variant="h5">{t('Content')}</Typography>
+                </Grid>
+                <Grid item>
+                  {datasetSchemaType !== SCHEMA_TYPES.NONE && (
+                    <Tooltip title={t('Dataset has Graasp schema')}>
```

Comment:
I would rather go for `follows Graasp Schema` but I am not sure maybe @hagoptaminian would know better.